<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stock Prediction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>42</y>
      <width>270</width>
      <height>256</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Symbol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lSymbol"/>
       </item>
       <item>
        <widget class="QPushButton" name="bAdd">
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="lTickers">
       <item>
        <property name="text">
         <string>PM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BUD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GIS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CVS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DIS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CRM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AAPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="bDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bDeleteAll">
         <property name="text">
          <string>Delete All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>109</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Prediction Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Lasso</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Linear</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start Period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cStartPeriod">
       <item>
        <property name="text">
         <string>2014</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2015</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2016</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2017</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2018</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="tTickerDisplay">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bPredict">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&amp;Predict</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>731</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tPredictions">
       <column>
        <property name="text">
         <string>Symbol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>X Split</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prev. Close</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prediction</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MAE ($)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Accuracy (%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>+/-</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>77</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="bSave">
       <property name="text">
        <string>&amp;Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
